tests/conftest.py

```diff
@@ -1190,9 +1190,14 @@ def mock_get_source_ip() -> Generator[_patch]:
         patcher.stop()
 
 
-@pytest.fixture(autouse=True, scope="session")
+@pytest.fixture(autouse=True, scope="module")
```

Comment:
We have this problem at runtime as well because we only load translations once in real production.  However I don't think we mock out integrations in production and than expect to use them latter.

Ideally we load the real translations before mocking the integration out so the cache doesn't get polluted with the mocks

Reply:
> Ideally we load the real translations before mocking the integration out so the cache doesn't get polluted with the mocks

Sadly we don't know which integrations will be doing the mocking, as it's totally unlinked to the place where it ends up failing. Here `tts` tests were causing `cloud` tests to fail.
I don't think we should rely on `tts` remembering to preload before mocking.

Reply:
Ideally we have a way to clear the cache after a run that uses `mock_integration`

Reply:
Ah - so turning `mock_integration` into a `@contextmanager`...
It's not really an undertaking I'm ready for right now but I can mark this PR as draft if someone else wants to have a go

Reply:
I think we cleanup the cache for the domain in `__exit__`, we only need to do it for this one test and we can deprecate calling it without a `contextmanager`, and slow migrate over.

Reply:
I think we can do this in a follow up, if we want to merge this PR now.